validate.py

```diff
@@ -17,7 +17,7 @@ def parse_args():
 
 
 def do_validate(c):
-    missing, unexpected = c.validate_all_present()
+    missing, unexpected, metadata_issues = c.validate_all_present()
```

Comment:
`data-owner-tools` doesn't generate metadata for household linkages yet. Will this report warnings for household linkage files? If so this should be noted for reviewers and we need to make sure it's fixed in a subsequent data-owner-tools ticket.